interpreter/src/main/scala/pl/touk/nussknacker/engine/Interpreter.scala

```diff
@@ -44,7 +44,19 @@ private class InterpreterInternal[F[_]](listeners: Seq[ProcessListener],
 
   private implicit def nodeToId(implicit node: Node): NodeId = NodeId(node.id)
 
-  private def handleError(node: Node, ctx: Context): Throwable => NuExceptionInfo[_ <: Throwable] = NuExceptionInfo(Some(node.id), _, ctx)
+  private def handleError(node: Node, ctx: Context): Throwable => NuExceptionInfo[_ <: Throwable] = {
+    val componentName = node match {
```

Comment:
We already have sth similar to this in `DefaultComponentIdProvider.nodeToComponentId`. It would be nice to have both things in one place

Reply:
`DefaultComponentIdProvider` needs `NodeData` from `engine.graph` package, while in my case `Node` is from `engine.compiledgraph`. So it looks similar but I'm not sure if I should put them in one place...

Reply:
I've extracted it into separate class in `interpreter` module.

Reply:
:+1: we can start with this approach and in the future think about some kind of merging both things. Can u add comment to this class that this logic should be kept synchronized with `DefaultComponentIdProvider` logic and TODO that it should be in one place in the future?

Reply:
TODO added :)